### ISSUE TITLE
[Enhancement]: A simple % discount field in the product editor?

### ISSUE THREAD

**Issue by loulemos** [woocommerce/woocommerce#37145]
### Describe the solution you'd like

I've used almost all the popular e-com platforms, all allow me to add a "% discount" when I'm editing a product. It seems this does not happen in Woo, you can do it only using the bulk editing option in the product search. To exemplify: 

**Wix**: [See image](https://i.imgur.com/Tar6XXH.png)

**Prestashop**: [See image](https://presta.site/blog/wp-content/uploads/2021/10/Screenshot_3-2.png)

**Shopify**: [See image](https://i.imgur.com/JZRJ66Y.png)

**Bigcommerce**: [See image](https://storage.googleapis.com/bigcommerce-production-dev-center/Knowledge%20Base/Promotions/Creating%20Product%20Discount%20Promotions/product-discount-setup.jfif)

But then in Woocommerce, you have this: [See image](https://i.imgur.com/RcQwIGv.png)

Should this be a priority? I'm working with a few different stores and every time I move someone to Woo this issue always comes up.

### Describe alternatives you've considered

Sure, there are plugins, but this should be included in Woo, since we already can apply a %.

**Comment by natko1** [comment]
Yes! I hope this gets implemented. Would make things so much easier. I was also surprised this wasn't a standard feature with WooComm. I hope an option for bulk rate setting is added to for variable products that have lots of variations, that way we don't need to edit each variation and can just set one discount % that will apply to all variations.

**Comment by natko1** [comment]
Still waiting. 🥱

**Comment by natko1** [comment]
@jonathansadowski any progress on this one?
